### PR TITLE
chore(docs): update landing and getting started page content

### PR DESCRIPTION
**Pull request by ayushmxn**
# Description

This is my attempt to update the docs. My plan is to open a PR for every page in the docs so changes can be reviewed easily. If anything is not to your liking, I can change it back.

### Added

- landing page: general intro + major features

### Changed

- mkdocs: fixed config
- getting started: simplify content

### Removed

- mkdocs: removed unused custom theme
- getting started: removed examples (~plan to add example usage in the API section itself~)

## Preview

Landing Page:
image

Getting Started Page:
image
image
image

### CODE REVIEW

File: docs/For Users/Getting Started.md [hunk 1]
```diff
@@ -1,167 +1,100 @@
...
+- Linux: `nw`
+- MacOS: `nwjs.app/Contents/MacOS/nwjs`
+- Windows: `nw.exe`
 
-See [the migration notes](Migration/From 0.12 to 0.13.md), if you are migrating your app from NW.js 0.12 or older versions.
+If you downloaded NW.js from `npm`, add it as a script:
 
-## Getting Help
-
-There are lots of useful information on [NW.js wiki](https://github.com/nwjs/nw.js/wiki). The wiki is also open for everyone, and you are encouraged to publish your knowledge on wiki.
```
Comment (rogerwang) [inline docs/For Users/Getting Started.md:163]: Should we keep the link to wiki?

File: docs/For Users/Getting Started.md [hunk 2]
```diff
@@ -1,167 +1,100 @@
...
-</html>
+```javascript
+nw.Window.open("./index.html", {}, () => {});
 ```
 
-You could also use the modules installed by [`npm`](https://www.npmjs.com/) with NW.js.
+More information can be found in the [API section](../References/App.md).
 
-!!! note "Native Node Modules"
-    Native Node modules, built when running `npm install`, are not compatible with NW.js ABI. To use them, you have to rebuild it from source code with [`nw-gyp`](https://github.com/nwjs/nw-gyp). See [Use Native Node Modules](Advanced/Use Native Node Modules.md) for details.
```
Comment (rogerwang) [inline docs/For Users/Getting Started.md:149]: Why is this removed?

File: docs/For Users/Getting Started.md [hunk 3]
```diff
@@ -1,167 +1,100 @@
...
-
-!!! tip "Drag &amp; Drop on Windows"
-    On Windows, you can drag the `folder containing package.json` to `nw.exe` to run your app.
-
-### Example 2 - Using NW.js APIs
-
-All NW.js APIs are loaded in `nw` object globally and can be used directly in JavaScript files. See [API References](../index.md#references) for full list of supported APIs.
+Here’s an example on how to use a JavaScript file as the entry point:
 
-This example shows how to create a native context menu in your NW.js app. You can create `index.html` with following content:
```
Comment (rogerwang) [inline docs/For Users/Getting Started.md:74]: The Menu example should be kept somewhere?

File: docs/For Users/Getting Started.md [hunk 4]
```diff
@@ -1,167 +1,100 @@
...
-  "name": "helloworld",
-  "main": "index.html"
+  "name": "nw-demo",
+  "main": "./index.html"
 }
 ```
 
-`package.json` is the manifest file in your app. It is written in [JSON format](http://www.json.org/). The `main` field figures out the first page opened by the NW.js if referenced to an HTML file, `"index.html"` in this example. And the `name` field is the unique name used among NW.js apps. See [Manifest Format](../References/Manifest Format.md) for more details.
-
-!!! tip "Use JS File as Main"
```
Comment (rogerwang) [inline docs/For Users/Getting Started.md:34]: This tip should not be removed?

File: docs/For Users/Getting Started.md [hunk 5]
```diff
@@ -1,167 +1,100 @@
...
-**Step 3.** Run your app
-
-```bash
-cd /path/to/your/app
-/path/to/nw .
-```
-
-`/path/to/nw` is the binary file of NW.js. On Windows, it's `nw.exe`; On Linux, it's `nw`; On Mac, it's `nwjs.app/Contents/MacOS/nwjs`.
-
-!!! tip "Drag &amp; Drop on Windows"
```
Comment (rogerwang) [inline docs/For Users/Getting Started.md:67]: And this tip as well.

File: docs/For Users/Getting Started.md (hunk 3 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:74]: There is already a Menu example in the API section - that's why I said to learn more check out the API reference.

File: docs/For Users/Getting Started.md (hunk 1 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:163]: Yup, done!

File: docs/For Users/Getting Started.md (hunk 2 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:149]: Added this back

File: docs/For Users/Getting Started.md (hunk 5 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:67]: Added back

File: docs/For Users/Getting Started.md (hunk 4 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:34]: Added back

File: docs/For Users/Getting Started.md (hunk 3 above)
Comment (ayushmxn) [reply docs/For Users/Getting Started.md:74]: My idea for the docs is include a complete example on how to use each API component under the definitions (taking inspiration from `esbuild`'s [docs](https://esbuild.github.io/api/#build-api)).
